db/src/bootstrap.rs

```diff
@@ -86,6 +90,26 @@ lazy_static! {
         ]
     };
 
+    static ref V1_CORE_SCHEMA: Vec<(symbols::NamespacedKeyword)> = {
```

Comment:
I don't understand how you came to this set.  Why isn't the full V1 schema the "core"?  Can you alter the core?  Can you alter things that aren't core?  (Nothing is going to work well if you alter anything in V1 -- there's an assumption throughout the code that the V1 schema is both present and currently asserted.)

Reply:
This is the full v1 schema… that is, these are all of the attributes that are present in the store after bootstrapping.

```
mentat=> .q [:find ?i :where [?a :db/valueType _][?a :db/ident ?i]]

:db/ident
:db/txInstant
:db.install/partition
:db.install/valueType
:db.install/attribute
:db/valueType
:db/cardinality
:db/unique
:db/isComponent
:db/index
:db/fulltext
:db/noHistory
:db.alter/attribute
:db/doc
:db.schema/version
:db.schema/attribute
```

Reply:
Idents don't affect vocabulary evolution; indeed, they're really a convenience. It's technically possible for an ident to exist in a store prior to a vocabulary that uses the name, and to continue to exist even if the vocabulary is 'removed'.